src/sections/Company/Brand/Brand-components/meshmap.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { SRLWrapper } from "simple-react-lightbox";
+import { Row, Col } from "../../../../reusecore/Layout";
+import Button from "../../../../reusecore/Button";
+import { FiDownloadCloud } from "@react-icons/all-files/fi/FiDownloadCloud";
+import MeshMapIconStacked from "../../../../assets/images/meshmap/meshmap-color-stacked.png";
+import MeshMapIconHorizontal from "../../../../assets/images/meshmap/meshmap-color-horizontal.svg";
+import MeshMapText from "../../../../assets/images/meshmap/meshmap-color-text.png";
```

Comment:
Thanks for this, @YarinDay. Will you consider only displaying the SVG versions?

Reply:
@leecalcote Yeah of course :), I'm on it!

Reply:
# Before
![image](https://user-images.githubusercontent.com/84711707/213878170-9137059c-bb87-4630-baed-681eb2d0b040.png)


# After
![image](https://user-images.githubusercontent.com/84711707/213878149-d2d2eed1-3efc-4398-b4fc-2d2c1979e53f.png)